build/ci-build.yml

```
@@ -74,6 +74,15 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
+          - task: AzureCLI@2
+            displayName: 'Get Token for Azure Management API'
+            inputs:
+              azureSubscription: XXXX
```

@tomkerkhove, can you provide the name of the service connection used for these tests so I can leverage it here to obtain a token?

---

We have this one but I don't know if it's used:
```suggestion
              azureSubscription: 'Azure - Codit-Arcus'
```

It should work AFAIK.